### PR TITLE
docs: Added the Japanese translation for /reference/configuration

### PR DESCRIPTION
**Pull request by nemuvski**
## Summary

I added the Japanese translation for the [biome configuration page](https://biomejs.dev/ja/reference/configuration/).

### CODE REVIEW

File: src/content/docs/ja/reference/configuration.mdx [hunk 1]
```diff
@@ -0,0 +1,1011 @@
...
+## `overrides`
+
+パターンのリストです。
+この設定を使用して、特定のファイルに対するツールの動作を変更します。
+ファイルがオーバーライドパターンに一致すると、そのパターンに指定された設定がトップレベルの設定を上書きします。
+パターンの順序は重要です。ファイルが3つのパターンに一致する場合、最初のパターンのみが使用されます。
+
+### `overrides.<ITEM>.ignore`
+
+Unixシェルスタイルのパターンのリストです。Biomeは、これらのパターンに一致するファイルに対してオーバーライドを適用しません。
```
Comment (chansuke) [inline src/content/docs/ja/reference/configuration.mdx:853]: `Unixシェル形式`

File: src/content/docs/ja/reference/configuration.mdx [hunk 2]
```diff
@@ -0,0 +1,1011 @@
...
+{
+  "overrides": [{
+    "ignore": ["scripts/*.js"]
+  }]
+}
+```
+
+### `overrides.<ITEM>.include`
+
+Unixシェルスタイルのパターンのリストです。Biomeは、これらのパターンに一致するファイルにのみオーバーライドを適用します。
```
Comment (chansuke) [inline src/content/docs/ja/reference/configuration.mdx:866]: same

**Review by chansuke — CHANGES_REQUESTED** [review]

File: src/content/docs/ja/reference/configuration.mdx [hunk 3]
```diff
@@ -0,0 +1,1011 @@
...
+    "ignore": ["scripts/*.js"]
+  }
+}
+```
+
+
+### `files.include`
+
+Biomeは、これらのパターンに一致するファイルやフォルダのみを処理します。
+Unixシェルスタイルのパターンのリストで指定します。
```
[the remark this answers is not in the document]
Comment (nemuvski) [reply src/content/docs/ja/reference/configuration.mdx:72]: Thank you for your review. I also thought that would be better.

File: src/content/docs/ja/reference/configuration.mdx [hunk 4]
```diff
@@ -0,0 +1,1011 @@
...
+  }
+}
+```
+
+パターン `src/**/*.js` に一致するファイルのみが処理され、パターン `scripts/**/*.js` に一致するファイルは無視されます。
+
+
+### `formatter.formatWithErrors`
+
+構文エラーがあるドキュメントをformatできるようにします。
```
[the remark this answers is not in the document]
Comment (nemuvski) [reply src/content/docs/ja/reference/configuration.mdx:363]: Thank you for pointing that out. The word `フォーマット` is automatically enforced as `format` by textlint, so I followed that rule.

However, in this case, the notation `フォーマット` is more appropriate. I will fix it. 🫡

File: src/content/docs/ja/reference/configuration.mdx (hunk 1 above)
Comment (nemuvski) [reply src/content/docs/ja/reference/configuration.mdx]: I fixed that. [c79413e](https://github.com/biomejs/website/pull/917/commits/c79413e54c5a9d7a7defc464356a5dcc12249eb3)

File: src/content/docs/ja/reference/configuration.mdx [hunk 5]
```diff
@@ -0,0 +1,1011 @@
...
+### `formatter.enabled`
+
+Biomeのformatterを有効にします。
+
+> デフォルト: `true`
+
+### `formatter.ignore`
+
+Biomeのformatterは、パターンに一致するファイルやフォルダを無視します。
+Unixシェルスタイルのパターンのリストで指定します。
```
[the remark this answers is not in the document]
Comment (nemuvski) [reply src/content/docs/ja/reference/configuration.mdx]: I fixed that. [c79413e](https://github.com/biomejs/website/pull/917/commits/c79413e54c5a9d7a7defc464356a5dcc12249eb3)

File: src/content/docs/ja/reference/configuration.mdx [hunk 6]
```diff
@@ -0,0 +1,1011 @@
...
+    "ignore": ["scripts/*.js"]
+  }
+}
+```
+
+
+### `linter.include`
+
+Biomeのlinterは、これらのパターンに一致するファイルやフォルダのみを処理します。
+Unixシェルスタイルのパターンのリストで指定します。
```
[the remark this answers is not in the document]
Comment (nemuvski) [reply src/content/docs/ja/reference/configuration.mdx]: I fixed that. [c79413e](https://github.com/biomejs/website/pull/917/commits/c79413e54c5a9d7a7defc464356a5dcc12249eb3)

File: src/content/docs/ja/reference/configuration.mdx [hunk 7]
```diff
@@ -0,0 +1,1011 @@
...
+### `linter.enabled`
+
+Biomeのlinterを有効にします。
+
+> デフォルト: `true`
+
+### `linter.ignore`
+
+Biomeのlinterは、パターンに一致するファイルやフォルダを無視します。
+Unixシェルスタイルのパターンのリストで指定します。
```
[the remark this answers is not in the document]
Comment (nemuvski) [reply src/content/docs/ja/reference/configuration.mdx]: I fixed that. [c79413e](https://github.com/biomejs/website/pull/917/commits/c79413e54c5a9d7a7defc464356a5dcc12249eb3)

File: src/content/docs/ja/reference/configuration.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (nemuvski) [reply src/content/docs/ja/reference/configuration.mdx]: I fixed that. [c79413e](https://github.com/biomejs/website/pull/917/commits/c79413e54c5a9d7a7defc464356a5dcc12249eb3)

File: src/content/docs/ja/reference/configuration.mdx (hunk 2 above)
Comment (nemuvski) [reply src/content/docs/ja/reference/configuration.mdx]: I fixed that. [c79413e](https://github.com/biomejs/website/pull/917/commits/c79413e54c5a9d7a7defc464356a5dcc12249eb3)

File: src/content/docs/ja/reference/configuration.mdx [hunk 8]
```diff
@@ -0,0 +1,1011 @@
...
+
+### `javascript.formatter.jsxQuoteStyle`
+
+JSXの文字列リテラルを表現する際に使用する引用符の種類です。 `"single"` または `"double"` のいずれかを指定できます。
+
+> デフォルト: `"double"`
+
+### `javascript.formatter.quoteProperties`
+
+オブジェクト内のプロパティが引用符で囲まれるべき時期を指定します。 `"asNeeded"` または `"preserve"` のいずれかを選択できます。
```
[the remark this answers is not in the document]
Comment (nemuvski) [reply src/content/docs/ja/reference/configuration.mdx]: I fixed that. [7893f11](https://github.com/biomejs/website/pull/917/commits/7893f115ccc2cf3509551f3b5894251dd4468760)

File: src/content/docs/ja/reference/configuration.mdx [hunk 9]
```diff
@@ -0,0 +1,1011 @@
...
+### `vcs.clientKind`
+
+VSCクライアントの種類を指定します。
+
+値：
+- `"git"`
+
+### `vcs.useIgnoreFile`
+
+BiomeがVCSの無視ファイルを使用するかどうかを指定します。`true` に設定すると、Biomeは無視ファイルに指定されたファイルを無視します。
```
[the remark this answers is not in the document]
Comment (nemuvski) [reply src/content/docs/ja/reference/configuration.mdx]: I fixed that. [0b9d86d](https://github.com/biomejs/website/pull/917/commits/0b9d86d42062ebdecff24006e7f61a47354bdf7e)

File: src/content/docs/ja/reference/configuration.mdx [hunk 10]
```diff
@@ -0,0 +1,1011 @@
...
+
+`ignore` と `include` を除いた、[上位のorganizeImports](#organizeimports) 設定のオプションが含まれます。
+
+### `overrides.<ITEM>.javascript`
+
+[上位のjavascript](#javascript) 設定のオプションが含まれます。
+
+#### Examples
+
+特定のフォルダ内のJavaScriptファイルのformat動作を変更することができます。
```
[the remark this answers is not in the document]
Comment (nemuvski) [reply src/content/docs/ja/reference/configuration.mdx]: i fixed that. [b223f81](https://github.com/biomejs/website/pull/917/commits/b223f8132b89823a5a8121ffe75d4866a0655daa)

File: src/content/docs/ja/reference/configuration.mdx (hunk 4 above)
[the remark this answers is not in the document]
Comment (nemuvski) [reply src/content/docs/ja/reference/configuration.mdx]: i fixed that. [790d93d](https://github.com/biomejs/website/pull/917/commits/790d93da1a8e42c17159d82ece22eb444d1a607f)

[the remark this answers is not in the document]
Comment (chansuke) [reply src/content/docs/ja/reference/configuration.mdx]: @nemuvski 
great!!

**Comment by chansuke** [comment]
@nemuvski 
Thank you for your work. it looks good to me.
If everything is complete, feel free to [Re-request](https://github.blog/changelog/2019-02-21-re-request-review-on-a-pull-request/)

**Review by chansuke — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**